packages/test/local-server-stress-tests/src/test/localServerStress.spec.ts

```diff
@@ -0,0 +1,108 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as path from "node:path";
+
+import {
+	type AsyncGenerator,
+	combineReducersAsync,
+	createWeightedAsyncGenerator,
+	takeAsync,
+} from "@fluid-private/stochastic-test-utils";
+
+import {
+	ddsModelMap,
+	DDSModelOpGenerator,
+	DDSModelOpReducer,
+	validateConsistencyOfAllDDS,
+	type DDSModelOp,
+} from "../ddsModels.js";
+import {
+	createLocalServerStressSuite,
+	LocalServerStressModel,
+	type LocalServerStressState,
+} from "../localServerStressHarness";
+import type { StressDataObjectOperations } from "../stressDataObject.js";
+
+import { _dirname } from "./dirname.cjs";
+
+type StressOperations = StressDataObjectOperations | DDSModelOp;
+
+const reducer = combineReducersAsync<StressOperations, LocalServerStressState>({
+	createDataStore: async (state, op) => {
```

Comment:
Here we have an async reducer calling a sync method (that could be meaningfully async) - is this a better place to float the promise, if desired?  Or maybe the promise should even be returned here?  You're making an AsyncReducer which seems like it wants to handle properly async methods.